rules/cloudflare_rules/cloudflare_firewall_high_volume_events_blocked.yml

```diff
@@ -2,12 +2,11 @@ AnalysisType: rule
 Filename: cloudflare_firewall_high_volume_events_blocked.py
 RuleID: Cloudflare.Firewall.HighVolumeEventsBlocked
 DisplayName: Cloudflare - High Volume Events Blocked
-Enabled: false
+Enabled: true
```

Comment:
action needed: 

This detection purports to detect high volume events, though the trigger for it is that the cloudflare firewall action is `block`.  
* We should confirm that the only reason for `block` is high-volume events.  Without that confirmation, I suspect we should not default this to `enabled: true` due to volume of alerts
* We should also tune the dedup() function for this. With the current title, any block event globally across all properties is going to suppress additional notifications. If multiple web properties were targeted at once, we'd group alerts via dedup, which might make the scale of the attack less clear.

Reply:
Flipping this to `false` for now while we update this detection in another branch.